src/sentry/uptime/subscriptions/regions.py

```diff
@@ -1,10 +1,16 @@
+from collections.abc import Sequence
+
 from django.conf import settings
 
+from sentry import options
 from sentry.conf.types.uptime import UptimeRegionConfig
 
 
 def get_active_region_configs() -> list[UptimeRegionConfig]:
-    return [v for v in settings.UPTIME_REGIONS if v.enabled]
+    configured_regions: Sequence[UptimeRegionConfig] = settings.UPTIME_REGIONS
+    disabled_region_slugs: Sequence[str] = options.get("uptime.disabled-checker-regions")
```

Comment:
May as well convert to a set?

Reply:
it's probably more expensive to convert to set here, let me see if it's possible to define options as sets

Reply:
Hmm.. I don't see us doing it in there anywhere

Reply:
It doesn't matter a lot, but we're iterating all the regions and checking the set multiple times, so it's probably more efficient?